File: src/ColecaoApp.Domain/Entities/Camisa.cs
using ColecaoApp.Domain.Enums;

namespace ColecaoApp.Domain.Entities;

public class Camisa
{
    public int Id { get; set; }
    public string NomeTime { get; set; } = string.Empty;
    public string Marca { get; set; } = string.Empty;
    public string Temporada { get; set; } = string.Empty;
    public Tamanho Tamanho { get; set; }
    public TipoCamisa Tipo { get; set; }
    public VersaoCamisa Versao { get; set; }
    public int? Numero { get; set;}
    public string? NomeJogador { get; set; }
    public bool IsAutografada { get; set; }
    public string? FotoUrl { get; set; }
    public Usuario Usuario { get; set;} = null!;
    public int UsuarioId { get; set; }
}

File: frontend/src/types/camisa.ts
export interface Camisa {
  id: number;
  nomeTime: string;
  marca: string;
  temporada: string;
  tamanho: number;
  tipo: number; 
  versao: number; 
  numero: number;
  nomeJogador: string;
  isAutografada: boolean;
  fotoUrl: string;
  usuarioId: number;
}

export interface AuthResponse {
  token: string;
}